File: src/main/js/FizzBuzz.js
var s = "";
for (var i = 1; i <= 100; i++) {
	if (i % 15 == 0) {
		s += "FIZZBUZZ ";
	} else if (i % 3 == 0) {
		s += "FIZZ ";
	} else if (i % 5 == 0) {
		s += "BUZZ ";
	} else {
		s += i + " ";
	} 
}
console.log(s);

File: src/main/java/FizzBuzz.java
public class FizzBuzz {
	public static void main(String [] argsv) {
		for (int i = 1; i <= 100; i++) {
			if (i % 15 == 0) {
				System.out.print("FIZZBUZZ "); 
			} else if (i % 3 == 0) {
				System.out.print("FIZZ ");
			} else if (i % 5 == 0) {
				System.out.print("BUZZ ");
			} else {
				System.out.print(i+" ");
			}
		}
		System.out.println();
	}

}
